CHANGELOG.md

```diff
@@ -12,6 +12,10 @@ and this project adheres to
 
 ### Changed
 
+- [#4913](https://github.com/firecracker-microvm/firecracker/pull/4913): Removed
```

Comment:
Not opposing the change, but what are customers supposed to do in response to this?

Reply:
Good point, they need to regenerate a snapshot. We might want to apply the same fix for the previous change.